From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arend-Jan van Hilten <a.j.vanhilten@tudelft.nl>
Date: Thu, 6 Feb 2025 13:50:45 +0000
Subject: rk3566-orangepi-3b disable download button audio jack
The download button on the Orange Pi 3B is connected to the audio jack microphone input.
Plugging in any 3 pin audio jack will trigger the download button, which is not desired as it will not boot.

Signed-off-by: Arend-Jan van Hilten <a.j.vanhilten@tudelft.nl>
---
 configs/orangepi-3b-rk3566_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/orangepi-3b-rk3566_defconfig b/configs/orangepi-3b-rk3566_defconfig
index 575dc4340d..c64f689975 100644
--- a/configs/orangepi-3b-rk3566_defconfig
+++ b/configs/orangepi-3b-rk3566_defconfig
@@ -94,5 +94,6 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_GENERIC=y
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC3_GENERIC=y
 CONFIG_ERRNO_STR=y
+CONFIG_ROCKCHIP_BOOT_MODE_REG=0
-- 
Created with Armbian build tools https://github.com/armbian/build

